src/main/java/project/model/AdminService.java

```diff
@@ -9,7 +9,7 @@
 public class AdminService {
     @RequestBody
```

Comment:
I think at this point we don't need this annotation anymore since we have same logic for each @StartMethod annotated method. I would check for input and for this annotation and that's it